app/admin/menu-items/new/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="new-item__form">
+{{#simple-form as |addItem|}}
```

Comment:
Here you are saying `as |addItem|` the block parameters are what the `simple-form` is sending to you. So, right now your inputs which are binding to `formValues` have no effect on the captured input.

Also this simple form does not have any `onsubmit` handler.